Samples/Docker-Simple/Build.cmd

```diff
@@ -0,0 +1,51 @@
+@echo off
+
+echo [7mTesting prerequisites[0m
```

Comment:
![image](https://user-images.githubusercontent.com/203839/32198899-52a9c0be-be1e-11e7-939d-d620e45816b3.png)
Are these color markers or some weird characters?

Reply:
Yes I tried to be a bit fancy here. This will only work with Windows 10. https://gist.github.com/mlocati/fdabcaeb8071d5c75a2d51712db24011#file-win10colors-cmd

Not sure why I didn't wrote this using powershell...